include/rocksdb/file_checksum.h

```diff
@@ -18,27 +18,42 @@
 
 namespace ROCKSDB_NAMESPACE {
 
-// FileChecksumFunc is the function class to generates the checksum value
+struct FileChecksumGenOptions {
+  std::string file_name;
+  std::string db_name;
```

Comment:
Is `db_name` used or needed?

Reply:
> Is `db_name` used or needed?

Thanks for the review. It have been removed. I double checked with the team, they just need file_name.